apps/server/src/modules/board/service/internal/board-context.service.spec.ts

```diff
@@ -5,26 +5,26 @@ import { Permission, RoleName } from '@shared/domain/interface';
 import { CourseRepo } from '@shared/repo/course';
 import { courseFactory, groupFactory, roleFactory, setupEntities, userFactory } from '@shared/testing';
 import { GroupTypes } from '@src/modules/group';
-import { RoomMemberService } from '@src/modules/room-member';
-import { roomMemberFactory } from '@src/modules/room-member/testing';
+import { RoomMembershipService } from '@src/modules/room-membership';
```

Comment:
as a general reminder, try to avoid `@src`. there is a ticket to remove it, but I dont know when it will be done...

Reply:
Ok, I overlooked this one. Fixed.